src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax;
+
+public abstract class Figure implements AreaCalculator, Drawable {
+    private Color color;
+    private TypeOfFigure typeOfFigure;
+
+    public Figure(Color color, TypeOfFigure typeOfFigure) {
+        this.setColor(color);
+    }

```

Comment:
this constructor should set color and figure type (not through setters, but directly)
change it please

Reply:
Can you tell me more information?
I can't understand what you want we to do.